src/lib/authselect_check.c

```diff
@@ -26,6 +26,43 @@
 #include "lib/authselect_files.h"
 #include "lib/authselect_private.h"
 
+static bool
+check_content(const char *content, const char *expected)
+{
+    const char *nextline;
+
+    /* Each generated file contains a "generated by" preamble that consists
+     * of two comment lines and one empty line. */
```

Comment:
Wouldn't it be easier to just have the preamble #defined and strcmp the preamble length? But meh..